scripts/check-representativeness.py

```
@@ -0,0 +1,243 @@
+    return {
+        "execs": [e.get("path", "") for e in (spec.get("execs") or [])],
+        "opens": len(opens),
+        "over_broad": [o.get("path", "") for o in opens
+                       if is_over_broad_open(o.get("path", ""))],
+        "ports": {p.get("name") for e in egress for p in (e.get("ports") or [])},
+        "ips": {e.get("ipAddress") for e in egress if e.get("ipAddress")},
+        "egress_count": len(egress),
+    }
+
+
+def check(name, rules, facts):
```

Ruff (0.16.0)

[warning] Too many branches (14 > 12)

(PLR0912)